plugins/output-kafka/src/main/scala/com/stratio/sparkta/plugin/output/kafka/KafkaOutputFormat.scala

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (C) 2016 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.stratio.sparkta.plugin.output.kafka
+
+object KafkaOutputFormat extends Enumeration {
+
+  type Status = Value
+  val ROW, JSON = Value
+
+  def toOutputFormat(format: String) : Value = format.toUpperCase match {
```

Comment:
Enumeration already implements this function: withName

http://stackoverflow.com/questions/1898932/case-objects-vs-enumerations-in-scala